backend/src/main/scala/bloop/CompilerCache.scala

```diff
@@ -309,7 +331,9 @@ final class CompilerCache(
         val invalidated = {
           zincManager match {
             case m: bloop.BloopClassFileManager => m.invalidatedClassFilesSet
-            case _ => zincManager.invalidatedClassFiles().toSet
+            // Bloop uses it's own classfile manager so this should not happen
+            case _ => new HashSet[File]()
```

Comment:
Shouldn't we at least log something here?

Reply:
Added logging